docs/reference/1-data-exchange/api-guidelines/design-and-implementation-guidelines/scope.md

```diff
@@ -0,0 +1,87 @@
+
+Unless otherwise stated, the guidelines provided by this document apply to both _compatible_ and _aligned_ API applications.
+
+## Ed-Fi API Specifications
+
+### Resource API
+
+The [Ed-Fi Unifying Data The [Ed-Fi Unifying Data Model](../../udm/readme.md) (UDM) provides the basis for the data transferred and manipulated by an Ed-Fi REST API implementation. The Ed-Fi UDM is a structured, conceptual model of common K–12 education data. The model includes entities that are easily recognized by educators and administrators: schools, students, teachers, attendance, grades, assessment results, and many others. These entities contain attributes (i.e., properties) that are also easily recognized. For example, assessment results contain data, such as a score and the date the assessment was administered. The UDM also includes associations (i.e., relationships) between entities, such as the association between students and schools.
+
+REST interfaces are built around Resources that define nouns. In the education domain, these nouns include such things as schools, students, and teachers. In the Ed-Fi UDM these nouns have been rigorously defined as "entities," with specific attributes and associations. Compositions of entities, with their attributes and associations, are called "domain aggregates." These are identified from the Ed-Fi UDM according to the principles of Domain-Driven Design (DDD). Domain aggregates are the Resources for an Ed-Fi REST API. These concepts are discussed in more detail later in this document.
+
+Entities are abstract concepts. They are implemented as "models" that define data types, validation requirements, and inter-relationships. The words are loosely interchangeable.
+
+_Also see_ [Resources](./api-design-guidelines/resources/readme.md).
+
+### Discovery API
+
+The Ed-Fi Discovery API presents clients with configuration information about a running application, including:
+
+* The software version.
+
+* Which version of the UDM is exposed by its implementation of the Resource API.
+
+* URL's for authentication and access to the various API specification(s) implemented by that application.
+
+* And, potentially, other runtime configuration parameters.
+
+The Discovery API allows client developers to build applications that need know only one base URL, can extract runtime information from it, and alter behavior as appropriate.
+
+_Also see_ [Discovery API](./api-design-guidelines/discovery-api.md).
+
+### Other API Specifications
+
+Ed-Fi API software _must_ utilize the OAuth 2.0 API specification for managing authentication, though the implementation _may_ use non-Ed-Fi software to provision OAuth-related services.
+
+:::tip
+
+The Ed-Fi ODS/API Platform exclusively uses a built-in OAuth 2.0 provider, which cannot be replaced by an off-the-shelf component. This need not be true of other implementations.
+
+:::
+
+Other specifications managed by the Ed-Fi Alliance include:
```

Comment:
Are we removing all instances of extra spacing in bullets? @stephenfuqua

Reply:
Not a strict requirement. I personally prefer not to have extra whitespace around each bulleted item, but I haven't been pushing the issue. This specific list seemed a little more egregious than others.